src/index.html

```diff
@@ -1,13 +1,292 @@
+
+    <main class="content">
+      <section id="about" class="content__first-section section">
+        <div class="section-left" >Move Free</div>
+        <div class="section-right">Shift your ride, not gears. Find the
+          fastest way to move in the city as the
+          bike adapts intuitively to power the speed
+          you need.
+        </div>
+      </section>
+
+      <section id="compare_bikes" class="content__second-section section">
+        <h2 class="section-center">Compare Bikes</h2>
+
+        <div class="wrapper">
+          <div class="wrapper__item">
+            <img
+              src="./styles/images/pictures/compare-bike-1.png"
+              alt="bike-image-compare"
+              class="wrapper__bike-img"
+            >
+            <div class="wrapper__bike-name">Sporty 4</div>
+            <article class="wrapper__bike-description">The iconic frame brought to a new performance height as a sporty, active ride.</article>
+            <div class="wrapper__bike-price">$ 2 590</div>
+          </div>
+          <div class="wrapper__item">
+            <img
+              src="./styles/images/pictures/compare-bike-2.png"
+              alt="bike-image-compare"
+              class="wrapper__bike-img"
+            >
+            <div class="wrapper__bike-name">Ride in town ST</div>
+            <article class="wrapper__bike-description">An open frame for an upright riding position as the most comfortable ride in town.
+            </article>
+            <div class="wrapper__bike-price">$ 2 090</div>
+          </div>
+          <div class="wrapper__item">
+            <img
+              src="./styles/images/pictures/compare-bike-3.png"
+              alt="bike-image-compare"
+              class="wrapper__bike-img"
+            >
+            <div class="wrapper__bike-name">Agile ride 3</div>
+            <article class="wrapper__bike-description">The lightweight frame that has earned its street tread as a sleek, agile ride.
+            </article>
+            <div class="wrapper__bike-price">$ 2 090</div>
+          </div>
+        </div>
+      </section>
+
+      <section id="details" class="content__third-section section">
+        <h2 class="section-center">The Details</h2>
+        <div class="details">
+          <div class="details-image">
+            <img
+            src="./styles/images/pictures/Details/1200/details_1200-1.png"
+            alt="picture-discover-app"
+            class="details-image-1"
+          >
+            <img
+            src="./styles/images/pictures/Details/1200/details_1200-2.png"
+            alt="picture-discover-app"
+            class="details-image-2"
+            >
+          </div>
+          <div class="details-title">Auto unlock</div>
+          <div class="details-text">
+            The app senses when you're nearby to unlock automatically.
+            GPS tracking so you know where your bike is and can track it anytime.
+          </div>
+        </div>
```

Comment:
Check your naming of classes. You have mistakes according ot BEM , you should write names of elements of the block details using __, not '-', as it looks so confusing, because previously you created elements adding __ and name of the element.